### PR TITLE
WIP: libmutter2 (and building budgie with it)

### PR DESCRIPTION
**Pull request by Cogitri**
This doesn't conflict with mutter 3.30 (or gnome-shell for that matter) and should allow for installing budgie and gnome-shell at the same time. @Gottox

### CODE REVIEW

**Comment by Cogitri** [comment]
Not quite sure if mutter&gnome-shell need a rev bump after reverting the revert

**Comment by Cogitri** [comment]
Also: Not runtime tested yet, time to get some sleep - should work though as nothing collides. Will test tomorrow

File: srcpkgs/libmutter2/template [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+ startup-notification-devel upower-devel"
+depends="desktop-file-utils zenity mutter"
+short_desc="Lightweight GTK+3 window manager"
+maintainer="Enno Boland <gottox@voidlinux.eu>"
+homepage="http://www.gnome.org"
+license="GPL-2.0-or-later"
+distfiles="${GNOME_SITE}/mutter/${version%.*}/mutter-${version}.tar.xz"
+checksum=8904236269295cd62c24cd1d7b64e1d0688fffe4c7386d620dcb342070cd036f
+wrksrc=mutter-$version
+shlib_provides="libmutter-clutter-2.so libmutter-cogl-2.so libmutter-cogl-pango-2.so"
```
[the remark this answers is not in the document]
Comment (Cogitri) [reply srcpkgs/libmutter2/template]: Eh? It doesn't provide libmutter-3.so though

**Comment by Gottox** [comment]
Be aware, that with my tests budgie-desktop was able to build, but crashed on startup.

**Comment by Cogitri** [comment]
> Seems like budgie-wm doesn't link correctly against libmutter-clutter-2.so for me right now though, will look into it soon

^ That's most likely why

File: srcpkgs/libmutter2/template (hunk 1 above)
[the remark this answers is not in the document]
Comment (maxice8) [reply srcpkgs/libmutter2/template]: oh derp, confused it with `Mutter`

**Comment by Cogitri** [comment]
OK, links correctly now but still doesn't start. However, even the Budgie from the repos (with mutter 3.28 installed) doesn't work for me...

**Comment by maxice8** [comment]
> Be aware, that with my tests budgie-desktop was able to build, but crashed on startup.

@Gottox i couldn't even start the version in the repos with either `sddm`, `xdm` or even by hand.

Seems like the package was broken and nobody was using it to begin with.

**Comment by Gottox** [comment]
I removed budgie-desktop from git and the repositories. We can update mutter and gnome-shell now.

**Comment by maxice8** [comment]
On Wed, Sep 12, 2018 at 04:14:02AM -0700, Enno Boland wrote:
> I removed budgie-desktop from git and the repositories. We can update mutter and gnome-shell now.
> 
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/2614#issuecomment-420611066

yay

**Pull request closed without merging**